docs/tools/tools_intro.md

```diff
@@ -27,9 +27,16 @@ Mbed OS 5 can be built with various toolchains. The currently supported versions
 
 ##### Arm Mbed Online Compiler
 
-The Mbed Online Compiler is our in-house IDE, and should be familiar to anyone who's been working with Mbed for a while. For more information, see the [Online Compiler page](developing-mbed-online-compiler.html).
+The Mbed Online Compiler is our in-house IDE, and should be familiar to anyone who's been working with Mbed for a while. It switches the backing toolchain based on the architecture version:
 
-For reference, the ARMC6 version the Online Compiler uses is 6.10.
+| Architecture version | Compiler |
+|----                  |----           |
+| Arm v6M       | Arm Compiler 5.06 |
+| Arm v7M       | Arm Compiler 5.06 |
```

Comment:
Can we please add full details on the compile version?
`Arm Compiler 5.06 Update 3`

Reply:
If I recall correctly, there was a Product decision earlier not to include the update version. Would you like us to revisit the topic?

Reply:
Yes please, let's discuss over email, can you loop me in please?
IMO we need to facilitate reproducibility of defects. The way to achieve this is by being transparent on dependencies, including tools. Thanks!

Reply:
I started typing this email but realized we list the update number above (and have for the past five releases), so I must be thinking of something else. I'm fine including the update numbers here.